cfgov/legacy/static/knowledgebase/less/es-ask-base.less

```diff
@@ -625,6 +625,63 @@ legend {
 }
 
 /* ==========================================================================
+   Autocomplete
+   ========================================================================== */
+@pacific:      #0072ce;
+@gray-20:      #d2d3d5;
+@gray-5:       #f7f8f9;
+@black:        #101820; 
+@block__bg:    @gray-5;
+
+.m-autocomplete {
+    position: relative;
+}
+
+.m-autocomplete_results {
+    box-sizing: border-box;
+    width: 100%;
+    max-height: 200px;
+    border: 1px solid #afafb3;
+    padding-left:  0;
+    overflow-y: scroll;
+    position: absolute;
+    z-index: 1000;
+    background-color: white;
+    list-style-type: none;
+    margin-top: 0;
+    font-family: "Avenir Next", Verdana, sans-serif;
+    background: #fff;
+    -webkit-box-shadow: 2px 3px 5px #afafb3;
```

Comment:
I think so. This is copied from the current knowledgebase autocomplete dropdown styles, so it should have the same behavior as the existing implementation.